apps/fyle/signals.py

```
@@ -0,0 +1,26 @@
+"""
+Fyle Signals
+"""
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from rest_framework.exceptions import ValidationError
```

Ruff (0.8.2)

`rest_framework.exceptions.ValidationError` imported but unused

Remove unused import: `rest_framework.exceptions.ValidationError`

(F401)